packages/rollup-full-node/src/app/handler/routing-handler.ts

```diff
@@ -0,0 +1,171 @@
+/* External Imports */
+import { Address } from '@eth-optimism/rollup-core'
+import { getLogger, logError, SimpleClient } from '@eth-optimism/core-utils'
+
+import { parseTransaction, Transaction } from 'ethers/utils'
+/* Internal Imports */
+import {
+  FullnodeHandler,
+  InvalidParametersError,
+  InvalidTransactionDesinationError,
+  Web3RpcMethods,
+} from '../../types'
+import { AccountRateLimiter } from '../utils'
+
+const log = getLogger('routing-handler')
+
+const methodsToRouteWithTransactionHandler: string[] = [
+  Web3RpcMethods.sendRawTransaction,
+  Web3RpcMethods.sendTransaction,
+  Web3RpcMethods.getTransactionByHash,
+  Web3RpcMethods.getBlockByNumber,
+  Web3RpcMethods.getBlockByHash,
```

Comment:
I'm not positive, but aren't these read only methods too?  Any reason these are sent to transaction handler?

Reply:
Yeah, they use in-memory state that is populated in the sendRawTransaction, so they have to be in the same process.